src/mgmt/srv_target.c

```diff
@@ -1010,9 +1010,11 @@ ds_mgmt_tgt_params_set_hdlr(crt_rpc_t *rpc)
 	D_ASSERT(in != NULL);
 
 	rc = dss_parameters_set(in->tps_key_id, in->tps_value);
-	if (rc == 0 && in->tps_key_id == DMG_KEY_FAIL_LOC)
+	if (rc == 0 && in->tps_key_id == DMG_KEY_FAIL_LOC) {
+		D_ERROR("Setting DMG_KEY_FAIL_VALUE = %"PRIu64"\n", in->tps_value_extra);
```

Comment:
This D_ERROR should be removed or turned into DEBUG message.

Reply:
will update